src/app/components/tile/index.html

```diff
@@ -65,7 +65,7 @@
       propertyName="settingsKey"
       isOptional="true"
     >
-    Specifies a unique key for the UI Config Service to retrieve stored settings from a database. You must prefix the key with the name of your SPA. The UI Config Service applies sticky settings to tile dashboards and saves configuration settings for users. This property accepts <sky-code>string</sky-code> values. This property is for internal Blackbaud use only. For Blackbaud developers, the UI Config Service website describes how to <a href="https://docs.blackbaud.com/ui-config-service-docs/learn/tutorials/save-tile-configuration">apply sticky settings to tile dashboards</a>.
+    Specifies a unique key for the UI Config Service to retrieve stored settings from a database. You must prefix the key with the name of your SPA. The UI Config Service saves configuration settings for users to preserve the layout and collapsed state of tile dashboards. This property accepts <sky-code>string</sky-code> values. This property is for internal Blackbaud use only. For Blackbaud developers, the UI Config Service website describes how to <a href="https://docs.blackbaud.com/ui-config-service-docs/learn/integrations/skyux-components/tiles">apply sticky settings to tile dashboards</a>.
```

Comment:
Same removal is necessary for the tile as well I believe.

Reply:
Done. Thanks for catching that. I'm going back and forth between 2 different PRs and missed it.